Client/core/CFilePathTranslator.cpp

```diff
@@ -80,7 +80,7 @@ void CFilePathTranslator::GetCurrentWorkingDirectory(std::string& WorkingDirecto
 
 void CFilePathTranslator::GetGTARootDirectory(std::string& ModuleRootDirOut)
 {
-    ModuleRootDirOut = g_gtaDirectory.u8string();
+    ModuleRootDirOut = g_gtaDirectory.string();
```

Comment:
leave it as u8string for utf8 compatibility

Reply:
See [comment](https://github.com/multitheftauto/mtasa-blue/pull/3636#discussion_r1729407241)